cmd/testdata/thresholds/non_existing_metric.js

```diff
@@ -0,0 +1,13 @@
+export const options = {
+	thresholds: {
+		// non existing is neither registered, nor a builtin metric.
+		// k6 should catch that.
+		"non existing": ["rate>0"],
```

Comment:
```suggestion
		"non_existing": ["rate>0"],
```
nit: I kind of prefer if this doesn't have a space in it - IIRC we don't enforce any character set on metrics, but we might want in the future and spaces might not be in that set then.